include/net/socket.h

```diff
@@ -838,12 +838,26 @@ struct ifreq {
 #define SOL_SOCKET 1
```

Comment:
For the commit "Extending posix....", please change the commit subject to
`net: socket: Extending Posix ......`